Cargo.toml

```diff
@@ -14,10 +14,12 @@ authors = ["Brendan Zabarauskas <bjzaba@yahoo.com.au>",
         "Simon Rönnberg",
         "Pierre Krieger",
         "Tomasz Stachowiak",
-        "Zeke Foppa"
+        "Zeke Foppa",
+        "Thomas O'Dell"
 ]
 license = "Apache-2.0"
 description = "A collision extension to cgmath"
+edition = "2018"
```

Comment:
we need a version bump of this library itself, given the breaking changes

Reply:
Agreed. Bumped to 20.